src/Moryx/Serialization/EntrySerializeSerialization.cs

```diff
@@ -120,6 +120,12 @@ private static IEnumerable<PropertyInfo> EntrySerializeAttributeFilter(Type sour
                 .Where(pm => pm.Mode.HasValue && pm.Mode == EntrySerializeMode.Never)
                 .Select(pm => pm.Property).ToArray();
 
+            //Class: Always
+            // if there are Always properties return those properties only
+            if ((sourceTypeMode.HasValue && sourceTypeMode.Value == EntrySerializeMode.Always) &&
```

Comment:
While this might fix the Resource error, it is a behavior change compared to the block below. Considering that the base class, already declares "Never", I propose that we only check for duplicate declaration of the attribute and give preference to the initial declaration, assuming that the base class should define the behavior.